zulipterminal/scripts/render_symbols.py

```diff
@@ -0,0 +1,60 @@
+import urwid
+
+from zulipterminal.config import symbols
+
+
+def exit_on_q(key: str) -> None:
+    if key in ('q', 'Q'):
+        raise urwid.ExitMainLoop()
+
+
+palette = [
+    ('header', 'light blue, bold', ''),
+    ('sub_header', 'white, bold', 'light blue'),
+]
+
+variables = vars(symbols)
+symbol_dict = {symbol: variables[symbol] for symbol in variables
+               if not symbol.startswith('__') and not symbol.endswith('__')}
+
+symbol_names_list = [urwid.Text(symbol, align='center')
+                     for symbol in symbol_dict]
+symbols_list = [urwid.Text(symbol_dict[symbol], align='center')
+                for symbol in symbol_dict]
+
+symbol_names_widget = urwid.LineBox(urwid.Pile(symbol_names_list),
+                                    title="Symbol Names",
+                                    title_attr='sub_header',
+                                    tlcorner='│', tline='─', lline='│',
+                                    trcorner='┬', blcorner='└', rline='│',
+                                    bline='─', brcorner='┴')
+symbols_widget = urwid.LineBox(urwid.Pile(symbols_list), title="Symbols",
+                               title_attr='sub_header',
+                               tlcorner='─', tline='─', lline='',
+                               trcorner='│', blcorner='─', rline='│',
+                               bline='─', brcorner='┘')
+
+symbols_display_box = urwid.Columns([
```

Comment:
This could be simpler if we wrap just Pile elements with this column widget.
(AFA I understand, the names are pretty straightforward, and so explicit titles of these names and symbols could be avoided. Let me know your thoughts on this :) )

Reply:
They are pretty straightforward, agreed, but I was of the opinion that an explicit mention would simplify things for users, but I suppose not using an explicit heading would remove the need for the nested LineBox, thereby simplifying the layout.